src/include/gurt/telemetry_common.h

```diff
@@ -47,11 +47,27 @@ enum {
 	D_TM_CLOCK_THREAD_CPUTIME	= 0x100,
 };
 
+struct d_tm_stats_t {
```

Comment:
Should we have doxygen comments for the structs and enums in these headers?

Reply:
I didn't see a lot of that happening in the DAOS header files.   I don't know if that's desirable or not.

Reply:
We've historically been pretty inconsistent about where we document public functions -- sometimes in the header file, sometimes in the .c file. The header is the only option for documenting structs though.